src/main/kotlin/hs/kr/equus/user/domain/auth/entity/repository/PassInfoRepository.kt

```diff
@@ -0,0 +1,9 @@
+package hs.kr.equus.user.domain.auth.entity.repository
+
+import hs.kr.equus.user.domain.auth.entity.PassInfo
+import org.springframework.data.repository.CrudRepository
+
+interface PassInfoRepository : CrudRepository<PassInfo, String> {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line